docs/user-guide/docs-source/spawn-with-houdini.md

```diff
@@ -2,15 +2,6 @@
 
 Ambit provides a tool to enable easy placement and randomization of your Houdini Obstacles. This tool is called **Spawn with Houdini** actor and operates very similarly to the [Spawn on Surface](/spawn-on-surface/) Ambit Spawner.
 
-## Enable Houdini Engine for Unreal Engine 4 Plugin
```

Comment:
Maybe include a note for install instructions similar to city-generation.md's line.

____
Is this step automatic for new projects?

Reply:
Yes. I tested it and if the Houdini Plugin is installed at the engine level then it is automatically enabled for new projects.